src/v2/view-builder/views/ov/SameDeviceEnrollLink.js

```diff
@@ -0,0 +1,51 @@
+import { View, _ } from '@okta/courage';
+import hbs from '@okta/handlebars-inline-precompile';
+import { FORMS as RemediationForms } from '../../../ion/RemediationConstants';
+
+export default View.extend({
+  className: 'ov-same-device-enroll-text',
+  template: hbs `
+        {{{i18n code="oie.enroll.okta_verify.select.channel.ovOnThisDevice"
+        bundle="login" $1="<a class='ov-same-device-enroll-link' href='#'>$1</a>"}}}
+      `,
+  postRender: function() {
+    const appState = this.options && this.options.appState;
+    const model = this.model;
+
+    this.$('.ov-same-device-enroll-link').click(function() {
+      if (!appState || !model) {
+        return;
+      }
+
+      const sameDeviceChannel = 'samedevice';
+      const remediations = appState.get('remediations');
+      const selectEnrollmentChannelRemediation = remediations.find((remediation) => {
+        return remediation.name === RemediationForms.SELECT_ENROLLMENT_CHANNEL;
+      });
+
+      if (!selectEnrollmentChannelRemediation) {
+        return;
+      }
+      const idField = _.find(selectEnrollmentChannelRemediation.uiSchema, (schema) =>
```

Comment:
Do we need to use `_.find()` and `_.filter()` throughout this file as opposed to `selectEnrollmentChannelRemediation.uiSchema.find()`?

Reply:
true I think we can just use `selectEnrollmentChannelRemediation.uiSchema.find()`, ill change it